### DISCUSSION TITLE
DTS directory name for MStar (mstar vs sigmastar)

### DISCUSSION THREAD

**Discussion by neuschaefer** [linux-chenxing/linux-chenxing.org#88] [General]
Hi!

The upstream kernel maintainers are currently discussing how to categorize the ARM devicetrees into subdirectories, because arch/arm/boot/dts has grown so large. During this, Arnd Bergmann commented that MStar devicetrees should be placed in `sigmastar`:

<https://lore.kernel.org/lkml/99b49e6b-e963-415a-a2c9-72505087833c@app.fastmail.com/>

Is this accurate?

**Comment by fifteenhex** [comment]
Thanks for bringing this to my attention. I didn't see the email.

I think mstar is probably better as that's where the platform originated (like freescale for some NXP chips). But I guess it doesn't matter too much. Anything is better than the current situation.
